x/group/server/testsuite/suite.go

```diff
@@ -1947,7 +1948,7 @@ func createProposal(
 
 	proposalRes, err := s.msgClient.CreateProposal(ctx, proposalReq)
 	s.Require().NoError(err)
-	return proposalRes.ProposalId
+	return group.ProposalID(proposalRes.ProposalId)
```

Comment:
let's just return `uint64` instead of `group.ProposalID` here, this will avoid back and forth conversions in the rest of the file.